### PR TITLE
Feature/blui 4199 contact screen base

### PR DESCRIPTION
**Pull request by JeffGreiner-eaton**
Fixes BLUI-4199.

#### Changes proposed in this Pull Request:

- add contact screen base
- add utilityClasses & slots with some test id's
- add contact screen base tests
- 

#### Screenshots / Screen Recording (if applicable)

image
image

#### To Test:

- yarn install:dependencies2 && yarn start:example2
- select contact us
- if you want to test the string alternative in example insert this to example with updated content in ContactScreenBase

emailSupportContent={(email): string => `Test for questions, feedback, or support please email us at ${email}.`} 
phoneSupportContent={(phone): string => `Test for technical support, please call ${phone}.`}

image

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: login-workflow/src/new-architecture/screens/ContactScreen/ContactScreenBase.tsx [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+ * @category Component
+ */
+
+const LinkStyles = {
+    fontWeight: 600,
+    textTransform: 'none',
+    textDecoration: 'none',
+    color: 'primary.main',
+    '&:visited': {
+        color: 'inherit',
```
Comment (ektaghag-eaton) [inline login-workflow/src/new-architecture/screens/ContactScreen/ContactScreenBase.tsx:32]: This CSS is not applying when making it **visited**.
Screenshot 2023-06-07 at 9 28 39 PM

**Review by ektaghag-eaton — COMMENTED** [review]
Other changes working perfectly fine 👍 .

File: login-workflow/src/new-architecture/screens/ContactScreen/ContactScreenBase.tsx (hunk 1 above)
Comment (JeffGreiner-eaton) [reply login-workflow/src/new-architecture/screens/ContactScreen/ContactScreenBase.tsx:32]: I see it does not work as its setup and will write an issue to add link styles visited in our theme repo

Comment (JeffGreiner-eaton) [reply login-workflow/src/new-architecture/screens/ContactScreen/ContactScreenBase.tsx:32]: https://github.com/etn-ccis/blui-react-themes/issues/74

**Review by ektaghag-eaton — APPROVED** [review]

**Pull request merged**